src/realmModels/Observation.js

```diff
@@ -397,6 +398,7 @@ class Observation extends Realm.Object {
         mapTo: "viewerTrustedByObserver",
         optional: true
       },
+      votes: "Vote[]",
```

Comment:
I just copied how the `faves` key is used here. I assume that syncing this key from the server to local is happening automatically. Because when searching for faves, I did not find any special handling.

Reply:
Yup, I think this works fine, though it does kind of duplicate the fave data here and in the `faves` property, so if you wanted to take it a step further you could remove `faves` and use `votes` to maybe replace it with a method like `faves()` that checks the `votes` property for fave votes. We'd be storing less data in Realm and fetching less data in requests. Definitely ok to merge as is, IMO, though.